components/nimbus/Cargo.toml

```diff
@@ -45,3 +46,4 @@ mockito = "0.31"
 env_logger = "0.7"
 clap = "2.33.3"
 tempfile = "3"
+glean = { path = "../external/glean/glean-core/rlb" }
```

Comment:
I don't think we need to include it in the dev-dependencies if it's already a dependency